File: Programmers/Lv.1/가장 가까운 같은 글자/142086.js
function solution(s) {
    const answer = [];
    
    for (let i=0; i<s.length; i++) {
        let num = -1;
        for (let j=0; j<i; j++) {
            if (s.charAt(i) === s.charAt(j)) {
                num = i - j;
            }
        }
        
        answer[i] = num;
    }
    
    return answer;
}

File: Programmers/Lv.1/가장 가까운 같은 글자/142086.java
class Solution {
    public int[] solution(String s) {
        int[] answer = new int[s.length()];
        
        for (int i=0; i<s.length(); i++) {
            int num = -1;
            for (int j=0; j<i; j++) {
                if (s.charAt(i) == s.charAt(j)) {
                    num = i - j;
                }
            }
            
            answer[i] = num;
        }
        
        return answer;
    }
}
